README.md

```diff
@@ -0,0 +1,4 @@
+## This is the Flatpak for OpenARENA, it uses Quake3e as its engine and has some bug fixes over the original version.
+- The wrapper script is configurable. Launch from the terminal and see the messages in red(when you launch for the first time).
+    - It allows you to load mods, or entirely different games based on Q3, you can also append new CVARS by simply adding +set at the end of the flatpak run command.
+#### NOTICE: If Q3e is unmaintained for more than 3 months counting from now, I will make a new Flatpak based on OmegA engine(I did not want to initially because I'd lose the ability to launch other games with the same flatpak, for example, UrbanTerror).
```

Comment:
```suggestion
#### NOTICE: If Q3e is unmaintained for more than 3 months counting from now, I will make a new Flatpak based on OmegA engine (I did not want to initially because I'd lose the ability to launch other games with the same flatpak, for example, UrbanTerror).
```